JobConfig/mixing/CreatePBISequence.fcl

```diff
@@ -12,6 +12,7 @@ services: {
 
 source: {
   module_type :  PBISequence
+  reconstitutedModuleLabel : PBISim
```

Comment:
I don't know why this shows up as a difference, this line is already merged.

Reply:
Yeah, I also noticed this, I am not sure